From 3880f7fea03b330f537cdd54c6c135e5e1c07dad Mon Sep 17 00:00:00 2001
From: Ethan Chen <intervigil@gmail.com>
Date: Mon, 24 Sep 2018 23:54:32 -0700
Subject: [PATCH 08/14] msm8974-common: Drop custom display flags

* TARGET_USES_OVERLAY and USE_OPENGL_RENDERER are largely deprecated
* Don't set triple buffering, it introduces latency
* Don't use custom VSYNC offsets

Change-Id: If2892c83726c3558a96b28a8404f49982ebb9ba0
[haggertk: Only triple offset aspect of this change is relevant to
 this device family]
---
 BoardConfigCommon.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 5b99e81..71fc86f 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -59,7 +59,6 @@ ifeq ($(HOST_OS),linux)
 endif
 
 # Display
-NUM_FRAMEBUFFER_SURFACE_BUFFERS := 3
 OVERRIDE_RS_DRIVER:= libRSDriver_adreno.so
 TARGET_ADDITIONAL_GRALLOC_10_USAGE_BITS := 0x02000000U
 
-- 
2.17.1

